src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx

```diff
@@ -374,9 +377,179 @@ For all alert conditions built with NRQL queries, the <DNT>**Signal history**</D
+2. Click <DNT>**Add a condition**</DNT>.
+
+## Copy a condition [#copy-condition]
+
+To copy an existing condition, including its [targets](/docs/apm/new-relic-apm/getting-started/glossary#alert-target) and [thresholds](/docs/apm/new-relic-apm/getting-started/glossary#alert-threshold), and add it to another policy for the selected account:
+
+1. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Alert conditions**</DNT>.
+2. From the list of alert conditions, click on the three dots icon of the alert you want to copy and select <DNT>**Duplicate condition**</DNT>.
+3. From the <DNT>**Copy alert condition**</DNT>, search or scroll the list to select the policy where you want to add this condition.
+4. Optional: Change the condition's name if necessary.
+5. Optional: Click the toggle switch to <DNT>**Enable on save**</DNT>
+6. Select <DNT>**Copy condition**</DNT>.
+
+    By default, the selected alert policy will add the copied condition in the **Disabled** state. Follow standard procedures to [add](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-alert-conditions) or copy more conditions to the alert policy, and then [**Enable** the condition](/docs/alerts/new-relic-alerts-beta/updating-alert-policies/disable-or-delete-alert-policies-conditions#disable-conditions) as needed. Additionally, the new condition will not copy any [tags](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/#add-via-ui-api) added to the original condition.
+
+## Configure an existing condition [#configure-condition]
+
+To change a condition, go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Alert Conditions**</DNT> and click a condition.
+
+* To [change the condition's name](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-alert-conditions#rename-condition) select <DNT>**Edit**</DNT> in the ellipses (...) menu for the condition in the list.
+* To update the values for the required Critical (red) or optional Warning (yellow) [thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-thresholds-trigger-alert), select <DNT>**Edit**</DNT> in the ellipses (...) menu for the condition in the list..
+* To update the condition's [runbook URL](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/provide-runbook-instructions-alerts), select <DNT>**Edit**</DNT> in the ellipses (...) menu for the condition in the list.
+
+## Enable/disable a condition [#disable-conditions]
+
+You can enable <img title="Alerts v3: Condition enabled" alt="041715icon-condition-on" src={accountsConditionOn}/> or disable <img title="Alerts v3: Condition disabled" alt="041715icon-condition-off" src={accountsConditionOff}/> any conditions, and the policy will continue to apply. To disable or re-enable a condition:
```

Comment:
What does this mean? `...and the policy will continue to apply`.  This doesn't make sense to me.

Reply:
You're right, that's not clear. I have removed it.